css/templates/land-see/page-vars.css

```diff
@@ -60,6 +60,12 @@
   --color-input-secondary: var(--color-font-secondary);
   --color-input-font-secondary: var(--color-font-secondary);
 
+  --color-accordion-color-base: #FDDCD7;
+  --opacity-100: 1;
```

Comment:
I dont think these need to be variables (also dont create bass style variables, it would be confusing when we build a configurator.)

Reply:
Makes sense.